CRM/ManualDirectDebit/Mail/Notification.php

```diff
@@ -73,19 +73,8 @@ private function setSenderEmailAddress($senderEmailAddress) {
    * @return bool
    */
   public function sendPaymentSignUpNotify($recurringContributionId) {
-    try {
-      $dataCollector = new CRM_ManualDirectDebit_Mail_DataCollector_RecurringContribution($recurringContributionId);
-      $tplParams = $dataCollector->retrieve();
-      $email = $dataCollector->retrieveEmail();
-      $messageTemplateId = CRM_ManualDirectDebit_Common_MessageTemplate::getMessageTemplateId(
-        CRM_ManualDirectDebit_Common_MessageTemplate::SING_UP
-      );
-
-      return $this->sendEmail($email, $messageTemplateId, $tplParams);
-    }
-    catch (CiviCRM_API3_Exception $e) {
-      return FALSE;
-    }
+    $dataCollector = new CRM_ManualDirectDebit_Mail_DataCollector_RecurringContribution($recurringContributionId);
+    return $this->sendEmail($dataCollector, CRM_ManualDirectDebit_Common_MessageTemplate::SIGN_UP_MSG_TITLE);
```

Comment:
Please add a blank line before the return statement. Also for the rest of notify functions.